spec/fixtures/sources/commonjs-grouped/g1.js

```diff
@@ -0,0 +1,5 @@
+const functions = require("../../../../src/index");
+
+exports.v1callable = functions.https.onCall((_data, _ctx) => {
```

Comment:
nit: if you're not going to use either parameter, why take any?

Reply:
I'm still a go programmer at heart :o